edXRegressionAutomated/Tests/Studio/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'mac'
```

Comment:
We should leave all the author stuff to the AUTHORS file, rather than on the individual files.